bin/localnet/Dockerfile-geth

```diff
@@ -0,0 +1,15 @@
+FROM frolvlad/alpine-glibc
+
+RUN apk add --no-cache curl
+#RUN apt-get update && apt-get -y install curl
+
+RUN mkdir -p /ethereum
+
+WORKDIR "/ethereum"
+VOLUME "/ethereum/keystore"
+VOLUME "/ethereum/geth-runtime"
+
+RUN curl https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.8.12-37685930.tar.gz > downloaded.tar.gz
+RUN tar --strip 1 -xvf downloaded.tar.gz && rm -rf downloaded.tar.gz
```

Comment:
We could merge these two commands via pipe, no need for extra file:

curl http://somefile.tar.gz | tar -zxf -

Reply:
And it worked. 💯 karma points!